ckan/tests/logic/test_validators.py

```diff
@@ -667,4 +667,31 @@ def call_validator(*args, **kwargs):
             errors[key] = []
             call_validator(key, {key: url}, errors, None)
 
+class TestOneOfValidator(object):
```

Comment:
These are great, thanks for adding them. You'll need to rewrite them to use the callable described above instead. Something like:

```python
def test_value_in_list(self):
    func = validators.one_of(['a', 'b', 'c'])
    assert_equals(func('a'), 'a')

def test_value_not_in_list(self):
    func = validators.one_of(['a', 'b', 'c'])
    raises_Invalid(func)('d')
```